sale_product_set_packaging_qty/__manifest__.py

```diff
@@ -3,14 +3,14 @@
 {
     "name": "Sale Product Set Packaging Quantity",
     "summary": "Manage packaging and quantities on product set lines",
-    "version": "13.0.1.0.1",
+    "version": "14.0.1.0.0",
     "development_status": "Alpha",
     "category": "Warehouse Management",
     "website": "https://github.com/OCA/sale-workflow",
     "author": "Camptocamp, Odoo Community Association (OCA)",
     "license": "AGPL-3",
     "application": False,
     "installable": True,
-    "depends": ["sale_product_set"],
+    "depends": ["sale_product_set", "sale_stock"],
```

Comment:
Why adding this dependency ? product.packaging is defined in product module and I don't see any requirements in code.

Reply:
Oh, didn't see above comments. Odoo has moved product_packaging (sale.order.line) fields to sale_stock...